.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile

```
@@ -106,6 +106,17 @@ pipeline {
                         }
                     }
                 }
+                stage("7.7.0") {
+                    agent {
+                        label 'linux'
+                    }
+                    steps {
+                        checkout scm
+                        script {
+                            runWith(lib, failedTests, "eck-76-${BUILD_NUMBER}-e2e", "7.6.2")
```

This is probably a copy-paste error?

```suggestion
                            runWith(lib, failedTests, "eck-77-${BUILD_NUMBER}-e2e", "7.7.0")
```

---

Yes.